src/app/rundown/components/full-segment-timeline/full-segment-timeline.component.ts

```diff
@@ -27,4 +24,8 @@ export class FullSegmentTimelineComponent {
   public setNextEvent: EventEmitter<RundownCursor> = new EventEmitter()
 
   public pixelsPerSecond: number = 30
+
+  public trackPart(_: number, part: Part): string {
```

Comment:
Is the `_: number` so you don't have to write `() => trackPart(part)` where you call the method?

If yes, then you have written your method for _how you intent to use it_ instead of _what does it need to do_, which is a bad habit/practice ;)

Reply:
I have tried doing that, but the Angular syntax for pipes do not allow for lambda functions in expressions 😞